cli/commands/project_create.go

```diff
@@ -0,0 +1,71 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/phase2/rig/cli/util"
+	"github.com/urfave/cli"
+)
+
+type ProjectCreate struct {
+	BaseCommand
+}
+
+func (cmd *ProjectCreate) Commands() []cli.Command {
+	create := cli.Command{
+		Name:        "create",
+		Aliases:     []string{},
+		Usage:       "Run a yeoman generator in a container to create project configuration files",
```

Comment:
Updated this to: "Run a code generator to generate scaffolding for a new project."